scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -254,43 +254,62 @@ it is useful e.g. when having multiple lanes with the same name, but with differ
   description = `changes the remote scope of a lane`;
-  extendedDescription = 'NOTE: available only before the lane is exported to the remote';
+  extendedDescription = `If no lane is specified (i.e. only one argument provided) this will update the current lane.
+NOTE: available only before the lane is exported to the remote`;
   alias = '';
-  options = [
-    [
-      'l',
-      'lane-name <lane-name>',
-      'the name of the lane to change its remote scope. if not specified, the current lane is used',
-    ],
-  ] as CommandOptions;
+  options = [] as CommandOptions;
   loader = true;
   migration = true;
+  example: [
+    {
+      cmd: 'lane change-scope my-org.other-scope';
+      description: 'change the scope of the current lane to my-org.other-scope';
+    },
+    {
+      cmd: 'lane change-scope my-specific-lane my-org.other-scope';
+      description: "change the scope of lane 'my-specific-lane' to my-org.other-scope";
+    }
+  ];
 
   constructor(private lanes: LanesMain) {}
 
-  async report([remoteScope]: [string], { laneName }: { laneName?: string }): Promise<string> {
-    const { remoteScopeBefore, localName } = await this.lanes.changeScope(remoteScope, laneName);
+  async report([laneNameOrRemoteScope, remoteScope]: [string, string]): Promise<string> {
+    const remoteScopeName = remoteScope ?? laneNameOrRemoteScope;
+    const laneToChange = remoteScope ? laneNameOrRemoteScope : undefined;
+    const { remoteScopeBefore, localName } = await this.lanes.changeScope(remoteScopeName, laneToChange);
     return `the remote-scope of ${chalk.bold(localName)} has been changed from ${chalk.bold(
       remoteScopeBefore
     )} to ${chalk.bold(remoteScope)}`;
   }
 }
 
 export class LaneRenameCmd implements Command {
-  name = 'rename <new-name>';
+  name = 'rename <existing-name-or-new-name> <new-name>';
   description = `EXPERIMENTAL. change the lane-name locally and on the remote (if exported)`;
+  extendedDescription =
+    'If no lane is specified (i.e. only a single argument provided) this will change the current lane';
   alias = '';
-  options = [
-    ['l', 'lane-name <lane-name>', 'the name of the lane to rename. if not specified, the current lane is used'],
-  ] as CommandOptions;
+  options = [] as CommandOptions;
   loader = true;
   migration = true;
+  example: [
+    {
+      cmd: 'lane rename new-lane-name';
+      description: 'rename the current lane to new-lane-name';
+    },
+    {
+      cmd: 'lane rename my-lane new-my-lane';
+      description: "change the scope of lane 'my-lane' to 'new-my-lane'";
+    }
+  ];
   constructor(private lanes: LanesMain) {}
 
-  async report([newName]: [string], { laneName }: { laneName?: string }): Promise<string> {
-    const { exported, exportErr, currentName } = await this.lanes.rename(newName, laneName);
+  async report([existingNameOrNewName, newName]: [string, string]): Promise<string> {
+    const newLaneName = newName ?? existingNameOrNewName;
```

Comment:
also this is not the right expected input it's 'laneName and newLaneName

Reply:
laneName is ambiguous, especially for a command that's changing the lane's name. So existingName sounds better to me